### DISCUSSION TITLE
Is it possible to define an uneven initial grid?

### DISCUSSION THREAD

**Discussion by ZhaofanZhu** [AMReX-Combustion/PeleC#689] [Q&A] [answered]
Hello, I am trying a wall boundary layer case. I want to make the initial mesh denser near the wall and sparser away from the wall. Can this be achieved? Thanks for your help!

**Comment by baperry2** [comment, marked as answer]
Yes. To refine a particular rectangular volume within your domain, use the following syntax in your input file:

```
tagging.refinement_indicators = yLow
tagging.yLow.max_level = 2          # Number of refinement levels to add in this part of the domain
tagging.yLow.in_box_lo = -0.1  -0.52  -0.85       # low (x,y,z) coordinate of refined region
tagging.yLow.in_box_hi =  3.1 -0.45    0.85        # high (x,y,z) coordinate of refined region
```

See `Exec/RegTests/ChannelFlow` for an example that is similar to what you want to do. You can also review the [documentation](https://amrex-combustion.github.io/PeleC/InputFiles.html#tagging-criteria) to see more information on all the different criteria that can be used to specify refinement.

Note that in the AMR framework used by PeleC, the base grid must always have fixed, uniform grid spacing with delta x = delta y = delta z, but this will add mesh refinement in the regions you are interested in.

**Reply by ZhaofanZhu** [reply]
Thank you very much! Mr.baperry2. It really works. 
The documentation is also detailed, I will read it carefully.
